libs/e2e/src/lib/describer/modelAssessment/modelOverview/ensureAllModelOverviewDatasetCohortsViewBasicElementsArePresent.ts

```diff
@@ -119,9 +119,21 @@ export function ensureAllModelOverviewDatasetCohortsViewBasicElementsArePresent(
         .first()
         .should("have.attr", "aria-label", expectedAriaLabel);
     }
+  } else if (datasetShape.isRegression) {
+    // 2 buttons and 1 hidden overflow button, 3 total
+    cy.get(Locators.ModelOverviewChartPivotItems).should("have.length", 3);
+    cy.get(Locators.ModelOverviewRegressionDistributionChart).should("exist");
+    cy.get(Locators.ModelOverviewProbabilityDistributionChart).should(
+      "not.exist"
```

Comment:
these kind of constants really should just be in one place... but I guess this is more of a general problem and out of scope for this PR